components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/api/dto/SObjectUrls.java

```diff
@@ -31,6 +31,7 @@
     private String compactLayouts;
     private String caseRowArticleSuggestions;
     private String push;
+    private String defaultValues;
```

Comment:
This one came back as a deserialization error when setting the Salesforce API version to 37.0.